Neos.Neos/Resources/Private/Templates/Error/Welcome.html

```diff
@@ -47,8 +47,8 @@ <h5>{neos:backend.translate(id: 'error.exception.{renderingOptions.renderingGrou
     </main>
   </div>
   <script>
-    $('.neos-error-details-toggle').click(function() {
-      $(this).toggleClass('neos-error-details-toggle--closed');
+    document.querySelector('.neos-error-details-toggle').addEventListener('click', function() {
+      document.querySelector('.neos-error-details-toggle').classList.toggle('neos-error-details-toggle--closed');
```

Comment:
```suggestion
    document.querySelector('.neos-error-details-toggle').addEventListener('click', function() {
      this.classList.toggle('neos-error-details-toggle--closed');
```

Reply:
Ping @sorenmalling

Reply:
@jonnitto I had a issue in firefox, where "this" wasn't nescessarily the toggle element, but the parent wrapper. Hence the "extra query"

Reply:
Hi guys,

sorry, did not find the time to go through my mentions.
In general, I like the usage of vanillajs. In this case, I would appreciate a more general solution.

Since version 7.1 we use vanillajs in many backend modules. For this extended the old window.NeosCMS API...
If this is not against you opinions, I would suggest making a general implementation for toggles so that we just need to add classes in the markup and the general BE code will take care. Otherwise, we will have multiple implementations again in the BE code.

I appreciate @sorenmalling initiative, just want to prevent a mess in some years :D

Reply:
@markusguenther Good idea. Or we could just include AlpineJS. And then we could create all kind of behaviour just in the HTML Markup

Reply:
Hmm but then we have again the dependency ;)

Reply:
Oh I missed that I already implemented a Dropdown and DropdownGroup for the User Module extension from @daniellienert ... so we just need to change the Markup already.

Will check that :)

Reply:
> need to add classes in the markup and the general BE code will take care.

I believe this is rendered as a exception - so I'm not fully aware if the backend code is present at this point?

Reply:
Ok maybe a bad naming from my side. With backend code I meant the JS stuff we use in the backend modules. If already added that and adjusted that a bit so that the components also work when we have no URIs for the configuration and translations.